website/docs/r/edgegateway.html.markdown

```diff
@@ -79,3 +79,34 @@ Default is `false`.
 `alert`, `critical`, `error`, `warning`, `notice`, `info`, `debug`
 
 
+## Importing
+
+Supported in provider *v2.5+*
+
+~> **Note:** The current implementation of Terraform import can only import resources into the state. It does not generate
+configuration. [More information.][docs-import]
+
+An existing edge gateway can be [imported][docs-import] into this resource via supplying its path. 
+The path for this resource is made of orgName.vdcName.edgeName. 
```

Comment:
I'd like us to use a consistent naming pattern of `org-name`, `vdc-name` and `edge-name` in the docs and comments.
Also, please remove the dot at the end of the sentence or shuffle the words in such a way that the path is not the last part of the sentence, so nobody thinks there's a dot at the end needed.